### PR TITLE
(WIP) ci(perf): add perf tracker script to publish perf stats

### PR DESCRIPTION
**Pull request by JoanFM**
**Proposal Proof of Concept for #619** 

@hanxiao @nan-wang would a skeleton like this make sense to have the idea under #619 implemented?

I was thinking about having a repo perf-tracker.git or whatever name that would get pushes to master from JINA BOT and whenever it does, it recomputes a graph with the performance history.

### CODE REVIEW

File: perf_tracker.sh [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+set -ex
+
+function run_perf_tracker {
+  for j in `seq 3`;
```
Comment (JoanFM) [inline perf_tracker.sh:6]: it should do some work to set it in csv format.

**Comment by JoanFM** [comment]
Yes, I guess it is possible, I guess you @hanxiao have permissions to allow another nightly build just for that purpose? Or maybe we can do all that in a new repo calles perf-tracker where every night we checkout jina, run the script, recompute the plot and pushes a new commit to itself?

**Comment by JoanFM** [comment]
I am concerned that we have no control on the machines and under which conditions they would run. So we would have some missleading information that often can be worse than no information at all

**Comment by nan-wang** [comment]
> I am concerned that we have no control on the machines and under which conditions they would run. So we would have some missleading information that often can be worse than no information at all

This is a good point. The machine provided by github might change from time to time. It is better to run it on the AWS machine.

**Comment by JoanFM** [comment]
> > I am concerned that we have no control on the machines and under which conditions they would run. So we would have some missleading information that often can be worse than no information at all
> 
> This is a good point. The machine provided by github might change from time to time. It is better to run it on the AWS machine.

I close this PR because of this!

**Pull request closed without merging**